src/backend/data/feed.js

```diff
@@ -76,7 +77,13 @@ class Feed {
    * Returns the newly created Feed's id as a Promise<String>
    */
   static async create(feedData) {
-    const feed = new Feed(feedData.author, feedData.url, feedData.etag, feedData.lastModified);
+    const feed = new Feed(
+      feedData.author,
+      feedData.url,
+      feedData.etag,
+      feedData.lastModified,
+      feedData.user
```

Comment:
Here you've got an ordering bug, for example.  The `user` needs to be third.

Reply:
fixed.